From 6ef25ff56a8ebd3d59c29f9dd99b4449564a28c8 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Thu, 1 Feb 2018 19:22:07 +0800
Subject: [PATCH 3305/5242] MLK-17490-2 drm/imx: dec400d: fix wrong path to
 define 'dcss_dec400d_write()'

commit  5c081ad7e8ea5ad8066b3a8fd492f9a09bf0ee56 from
https://source.codeaurora.org/external/imx/linux-imx.git

The macro 'USE_CTXLD' usage in function 'dcss_dec400d_write()'
is opposite to the real defintion path.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dcss/dcss-dec400d.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/imx/dcss/dcss-dec400d.c b/drivers/gpu/imx/dcss/dcss-dec400d.c
index acd1221..435cc77 100644
--- a/drivers/gpu/imx/dcss/dcss-dec400d.c
+++ b/drivers/gpu/imx/dcss/dcss-dec400d.c
@@ -50,7 +50,7 @@ static void dcss_dec400d_write(struct dcss_dec400d_priv *dec400d,
 			       uint32_t value,
 			       uint32_t offset)
 {
-#if USE_CTXLD
+#if !USE_CTXLD
 	dcss_writel(value, dec400d->dec400d_reg + offset);
 #else
 	dcss_ctxld_write(dec400d->dcss, dec400d->ctx_id,
-- 
1.7.9.5

